### DISCUSSION TITLE
when ssr, i want to intercept 500 status and do something，then response to user。

### DISCUSSION THREAD

**Discussion by lijinghust** [nuxt/nuxt#21004] [Questions]
### Describe the feature

when ssr, i want to intercept 500 status and do something，then response to user。

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by xjccc** [comment]
use [interceptors](https://nuxt.com/docs/api/composables/use-fetch#example)
